server/service/async/async_label_test.go

```diff
@@ -0,0 +1,392 @@
+package async
```

Comment:
This is just label-specific test code that got moved to a new file for readability.